components/chat/chatBar/upload/files.tsx

```diff
@@ -47,8 +47,9 @@ const Files: React.FC<FilesProps> = ({ files, setFiles }) => {
                   color="primary"
                   radius="full"
                   onPress={async () =>
-                    window.alert(
-                      'Coming soon, we promise! This will open your file.'
+                    // This submits a request to the electron main process to open the file
+                    (window as any).electronAPI.openFile(
```

Comment:
Can you globally update the window object to add TypeScript support?

ideally add an `@/types/window.d.ts` with these contents:

```ts
declare global {
  interface Window {
    electronApi: <ElectronApiType>;
  }
}

export {} 
```